docs/03-for-team-members/02-becoming-a-team-member.md

```diff
@@ -0,0 +1,48 @@
+# Team members
+
+Flathub, as a centralized repository of software apps, has a dedicated team of
+contributors necessary to keep it running. These contributors are in charge of:
+
+* Reviewing new applications
+* Reviewing permission changes for existing aplications
+* Requesting permission changes for existent apps that might not fit the
+  guidelines
+
+In consequence, they can be seen as the auditors and moderators of our
+community. As such, they play a big role in the decision-making process.
+They are also an important part of the Flathub and Flatpak trust model. The fact
+that applications' permissions are audited by the community is a helps to ensure
+consistent and safe permissions in the majority of the cases. Therefore, they
+also enjoy the recognition for their work!
+
+## Becoming a team member
+
+Since auditing apps and permissions is a great task, we are always in need of
+more hands. If you want to be part of our team, help out and enjoy the benefits,
+simply follow these steps:
+
+* Engage with the community, so that other team members can know you.
+* Help review [pending applications](https://github.com/flathub/flathub/pulls)
+  so that they match the
+  [requirements](https://docs.flathub.org/docs/for-app-authors/requirements).
+  You don't need to be a team member to do so, more eyes are always welcomed,
+  and that way you can prove your interest and ability to do the reviews.
+  As with everything, there is always a learning curve, so the recommendation is
+  to start small and listen to the feedback of existing team members.
+  TODO: For how long should have people contributed? How many apps should they
+  have reviewed?
```

Comment:
I think not putting any limit is better. The current team can decide on case by case basis.